cmake/FindMyGUI.cmake

```diff
@@ -109,6 +109,13 @@ ELSE (WIN32) #Unix
     ENDIF(MYGUI_STATIC)
 ENDIF (WIN32)
 
+if (ANDROID)
```

Comment:
There is already a find_package freetype in the same file, on line 89. Why not use that one?